invenio_cli/commands/upgrade.py

```diff
@@ -52,3 +52,33 @@ def upgrade():
         ]
 
         return steps
+
+    @staticmethod
+    def update_app(version):
+        """Update dependencies..
+
+        It is a class method since it does not require any configuration.
+        """
+        prefix = ['pipenv']
+        app = prefix + [
+            'install',
+            'invenio-app-rdm[postgresql, elasticsearch7]~=' + version
+        ]
+        dependencies = prefix + [
+            'update'
+        ]
```

Comment:
Running `pipenv install invenio-app-rdm[postgresql, elasticsearch7]~=<nvew version>` and then `pipenv update` gave the same result as just running `pipenv install invenio-app-rdm[postgresql, elasticsearch7]~=<nvew version>` only. Am I missing something or is `pipenv update` redundant here?

Reply:
I thought that pipenv update was needed to keep all the dependencies updated, I will remove that command.